dev/image/Dockerfile

```diff
@@ -10,7 +10,8 @@ ENV WORKSPACE_KERNEL 5.4.0-1033-gke
 USER root
 
 ### QEMU (x86) and workspace kernel development tools
-RUN install-packages qemu qemu-system-x86 linux-image-$WORKSPACE_KERNEL libguestfs-tools sshpass
+RUN install-packages qemu qemu-system-x86 linux-image-$WORKSPACE_KERNEL libguestfs-tools openssh-sftp-server
```

Comment:
When this file gets changed I'd expect to see updated `dev` images in the `.gitpod.yml` and similar places (search and replace). Is that omission intentional?

Reply:
I just didn't do it yet, thanks for the reminder :)

Reply:
The reason is that the branch needs to be pushed before the image gets pushed to gcr, so I was waiting for the CI to do its thing

Reply:
Did it now!